ClientApp/viking-vault/src/UserDashboard/AutomaticPaymentList.tsx

```diff
@@ -81,6 +82,16 @@ class AutomaticPaymentList extends React.Component<any, IAutomaticPaymentsState>
         this.getAutomaticPayments();
     }
 
+    deletePaymentFromList = (id : number) =>{
+        let paymentList = this.state.payments.filter(payment => {
```

Comment:
Opposite in here, sometimes It makes more sense to use one liner's for simple methods such as `.filter()` and single `setState`. 

Just try think at all times that a new person might join your team and would have to read your code and put yourself in their shoes, how would would you want the code to be written in such a way that it would make their life easier?...

`let paymentList = this.state.payments.filter(payment => payment.id !== id)`

`this.setState({ payments : paymentList });`